docs/rims.md

```diff
@@ -0,0 +1,203 @@
+# Reference Integrity Manifests
+
+Trusted Computign Group (TCG) specifies Reference Integrity Manifest (RIM) as a
+structure to convey information about the software and firmware running on
+given system. These manifests are used as an input to retrieve the reference
+golden measurements which indicate teh system is genuine, trusted and runs a
+configuration and software approved by the RIM issuer (System Supplier, Value
+Added Reseller, Maintenance Organisation, etc.).
+
+![RIMs](/images/rims.png)
```

Comment:
I have used absolute paths on purpose. When the doc would be migrated, the links would break. Now they refer to the mkdocs root directory which is `/docs` directory in the repo, so even when the doc moves, the links will work (unless the images directory will change).